### PR TITLE
made a change

### PR DESCRIPTION
**Pull request by east-empire**

### CODE REVIEW

File: spec/ClearCheck.spec.js [hunk 1]
```diff
@@ -32,4 +32,6 @@ describe('the ClearCheck.js class', () => {
 
 		expect(result).toBeDefined();
 	});
+
+	// making changes
```
Comment (mrmarcsmith) [inline spec/ClearCheck.spec.js:36]: Why did we add this comment?

**Review by mrmarcsmith — CHANGES_REQUESTED** [review]
see notes

File: spec/ClearCheck.spec.js (hunk 1 above)
Comment (east-empire) [reply spec/ClearCheck.spec.js:36]: because i said so.

**Review by mrmarcsmith — APPROVED** [review]

**Comment by nusa232** [comment]
lgtm

File: spec/ClearCheck.spec.js [hunk 2]
```diff
@@ -32,4 +32,6 @@ describe('the ClearCheck.js class', () => {
 
 		expect(result).toBeDefined();
```
Comment (nusa232) [inline spec/ClearCheck.spec.js:33]: why is that expected to be defined :P ?

Comment (east-empire) [reply spec/ClearCheck.spec.js:33]: because the function returns a value. see src.

**Review by nusa232 — APPROVED** [review]

**Pull request merged**